### ISSUE TITLE
ICICI bank home page! ‘the next button is not label properly.

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#11]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.press  H for heading till  reached to the “Coral Credit Card ‘ heading level 3  
3.pressed B for button to reached ‘next button’
expected Result .’next button ‘should be announced the purpose of the button
Actual Result the screen reader announced only ‘next button ‘
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
